about.html

```diff
@@ -50,7 +50,7 @@ <h3 class="title">Get In Touch</h3>
                         <h3>Quick Links</h3>
                         <ul>
                             <li><a class="colored" href="about.html">About Us</a></li>
-                            <li><a class="colored" href="https://google.com/">Services</a></li>
```

Comment:
Yes, it wasn't supposed to be google.com

Reply:
Thank you, sir. I am really impressed and grateful that you acknowledge me. Thank you very much, sir.